### ISSUE TITLE
x86 x64 processor support

### ISSUE THREAD

**Issue by OmlineEditor** [Lucchetto/SuperImage#29]
please add support not only for mobile processors, but also for desktop x86 x64

**Comment by Lucchetto** [comment]
Is there an x86 Android device with Vulkan or OpenCL support ? I excluded them because for that reason

**Comment by OmlineEditor** [comment]
I do not know about the support, you know better. can you study this question in more detail?

**Comment by OmlineEditor** [comment]
I know some algorithms can work without the support of a video adapter only using a processor. therefore, there should be no problems with x86 support. will work differently on the processor

**Comment by Lucchetto** [comment]
Yes everything be executed on CPU but it would many times slower

**Comment by OmlineEditor** [comment]
> Yes everything can be executed on CPU but it would many times slower

I know, but it's not a problem if there is an x86 processor on the computer in the emulator. there is no battery that runs out and the processor on the computer is quite powerful

**Comment by mattdrepo** [comment]
Came along to bump this thread for a few reasons:

Bliss OS is trying and has tackled initial Vulkan Support

"Out of RAM" issues can be fixed by adding more ram

More powerful CPU's will tackle the issue of CPU only tasks (server grade)

Please consider, because even if these use-cases aren't on the X86 builds, low power machines can perform the tasks without using battery, and under 'undervolt'/low-wattage conditions.

Also consider for every use case a TERMUX version or batch process.  For creative and large tasks.

Congratulations for a great project overall, and this is no means criticism, just another upvote for X86 64

https://www.xda-developers.com/bliss-os-android-rom-desktop-pc-vulkan-graphics-api-support/
